app/enterprise/changelog.md

```diff
@@ -6,71 +6,215 @@ layout: changelog
+    the Admin API endpoints for entities (such as `/routes` and
+    `/services`) are read-only, since the configuration can only
+    be updated via `/config`
+- Admin API endpoints now support searching by tag
+  (for example, `/consumers?tags=example_tag`)
+  - You can search by multiple tags:
+     - `/services?tags=serv1,mobile` to search for services matching tags `serv1` and `mobile`
+     - `/services?tags=serv1/serv2` to search for services matching tags `serv1` or `serv2`
+- New Admin API endpoint `/tags/` for listing entities by tag: `/tags/example_tag`
+
+##### PDK
+
+- New function `kong.node.get_memory_stats()`. This function returns statistics
+  on shared dictionaries and workers Lua VM memory usage, and powers the memory
+  statistics newly exposed by the `/status` endpoint.
+- New PDK function: `kong.client.get_protocol` for obtaining the protocol
+  in use during the current request
+- New PDK function: `kong.nginx.get_subsystem`, so plugins can detect whether
+  they are running on the HTTP or Stream subsystem
+
+#### Plugins
+
+- Logging plugins: log request TLS version, cipher, and verification status.
+- Plugin development: inheriting from `BasePlugin` is now optional. Avoiding
+  the inheritance paradigm improves plugins' performance.
+- Support for ACL **authenticated groups**, so that authentication plugins
+  that use a 3rd party (other than Kong) to store credentials can benefit
+  from using a central ACL plugin to do authorization for them.
+- The Kubernetes Sidecar Injection plugin is now bundled into Kong for a 
+  smoother K8s experience.
+- AWS Lambda now includes the AWS China region.
+
+#### CLI
+
+- **Bulk database import** using the same declarative
+  configuration format as the in-memory mode, using the new command:
+  `kong config db_import kong.yml`. This command upserts all
+  entities specified in the given `kong.yml` file in bulk
+- New command: `kong config init` to generate a template `kong.yml`
+  file to get you started
+- New command: `kong config parse kong.yml` to verify the syntax of
+  the `kong.yml` file before using it
+- New option `--wait` in `kong quit` to ease graceful termination when using orchestration tools.
+
+### Fixes
+
 #### Core
 
-- Adds support for [`db_cache_warmup_entities`](/1.2.x/configuration/#db_cache_warmup_entities), 
-which allows Kong to pre-load all necessary entries into Kong nodes' memory on start.
+- Resolve hostnames properly during initialization of Cassandra contact points
+- Fix health checks for Targets that need two-level DNS resolution
+  (e.g. SRV → A → IP)
+- Fix serialization of map types in the Cassandra backend
+- Fix target cleanup and cascade-delete for Targets
+- Avoid crash when failing to obtain list of Upstreams
+- Disallow invalid timeout value of 0ms for attributes in Services
+- DAO fix for foreign fields used as primary keys
+- Fixes a memory usage growth issue in the `/config` endpoint when configuring
```

Comment:
This can also be removed as not applicable to enterprise.

Reply:
Thanks, @RobSerafini --just to confirm, remove line 183 only?